### PR TITLE
feat: alt + arrow key to move cursor one word

### PR DESCRIPTION
**Pull request by Xazin**
Relates: https://github.com/AppFlowy-IO/AppFlowy/issues/2071

### CODE REVIEW

**Comment by Xazin** [comment]
@LucasXu0 When you have time, can you guide me on how to resolve the issue with tests. I had to make changes because I took over the command that the failing tests were taking advantage of.

File: test/service/shortcut_event/shortcut_event_test.dart [hunk 1]
```diff
@@ -117,15 +118,16 @@ void main() async {
...
+            linuxCommand: 'ctrl+arrow right',
+            macOSCommand: 'ctrl+arrow rightt',
           );
         }
       }
+
       await editor.pressLogicKey(
         LogicalKeyboardKey.arrowRight,
-        isAltPressed: true,
+        isControlPressed: true,
```
Comment (a-wallen) [inline test/service/shortcut_event/shortcut_event_test.dart:130]: `appflowy_flutter\packages\appflowy_editor\lib\src\service\shortcut_event\key_mapping.dart` must not have an entry that handles this particular combination of keys which is why the test is failing.

File: test/service/shortcut_event/shortcut_event_test.dart [hunk 2]
```diff
@@ -117,15 +118,16 @@ void main() async {
       for (final event in builtInShortcutEvents) {
         if (event.key == 'Move cursor end') {
           event.updateCommand(
-            windowsCommand: 'alt+arrow right',
-            linuxCommand: 'alt+arrow right',
-            macOSCommand: 'alt+arrow right',
+            windowsCommand: 'ctrl+arrow right',
+            linuxCommand: 'ctrl+arrow right',
+            macOSCommand: 'ctrl+arrow rightt',
```
Comment (LucasXu0) [inline test/service/shortcut_event/shortcut_event_test.dart:123]: Hey, @Xazin. Double `t` here.

**Review by MayurSMahajan — COMMENTED** [review]
In `test/service/shortcut_event/shortcut_event_test.dart` file when you are updating the command to ctrl+arrow left, in macOSCommand you want to use **meta**  instead of ctrl. On line 66: `macOsCommand: 'meta+arrow left',`

File: test/service/shortcut_event/shortcut_event_test.dart (hunk 2 above)
Comment (Xazin) [reply test/service/shortcut_event/shortcut_event_test.dart]: Thanks

File: test/service/shortcut_event/shortcut_event_test.dart (hunk 1 above)
Comment (Xazin) [reply test/service/shortcut_event/shortcut_event_test.dart:130]: This is actually done on purpose, I didn't actually touch anything that _should_ have changed this test from succeeding, but apparently a few lines above in the test, it "updates" a command, and the key events used to trigger it, which was ones I had used in the new command implemented in this feature, thus it failed due to the colliding commands, hence the change here.

**Comment by Xazin** [comment]
Thanks @LucasXu0 - That was the issue ...

**Pull request merged**